crates/rolldown_resolver/src/resolver.rs

```diff
@@ -157,26 +157,19 @@ impl<F: FileSystem + Default> Resolver<F> {
     };
     let resolution = if let Some(importer) = importer {
       let context = importer.parent().expect("Should have a parent dir");
-      selected_resolver.resolve(context, specifier)
+
+      if context.to_str().map_or(true, str::is_empty) {
```

Comment:
Here is better to add some notes for catching virtual files.

Reply:
done